checkmackerelmetric/checkmackerelmetric.go

```diff
@@ -85,7 +87,17 @@ func (opts *mackerelMetricOpts) run() *checkers.Checker {
 
 	conf, err := config.LoadConfig(config.DefaultConfig.Conffile)
 	if err != nil {
-		return checkers.Unknown(fmt.Sprintf("%v", err))
+		if runtime.GOOS == "windows" {
+			newpath := filepath.Join(config.DefaultConfig.Conffile, "../../../mackerel-agent.conf")
+			conf, err = config.LoadConfig(newpath)
+			if err != nil {
+				return checkers.Unknown(fmt.Sprintf("%v", err))
```

Comment:
I think the following kind of code would also be good.
```suggestion
				return checkers.Unknown(err.Error())
```

Reply:
Thanks! I'll apply it to other projects... 😻